lib/cli.js

```diff
@@ -12,13 +12,15 @@ var Path = require('path');
 
 var USAGE = [
     'Usage: json5 -c path/to/file.json5 ...',
-    'Compiles JSON5 files into sibling JSON files with the same basenames.',
+    '    Compiles JSON5 files into sibling JSON files with the same basenames.',
+    'or:  json5 --validate path/to/file.json5 ...',
+    '    Validates JSON5 files'
 ].join('\n');
 
-// if valid, args look like [node, json5, -c, file1, file2, ...]
+// if valid, args look like [node, json5, -c || -t, file1, file2, ...]
 var args = process.argv;
 
-if (args.length < 4 || args[2] !== '-c') {
+if (args[2] === '-c' || args[2] === '--validate' && args.length < 4) {
```

Comment:
Also, is this boolean logic right? I thought `&&` takes precedence over `||`, so you might need to wrap the `-c || --validate` check in parentheses.